lib/ecto/migration.ex

```diff
@@ -740,7 +740,7 @@ defmodule Ecto.Migration do
   end
 
   @doc """
```

Comment:
```suggestion
  @doc ~S"""
```

Reply:
Right, but not needed now. `\#{key}` was used for bindings in old `dynamic/2` implementation, so there is no longer need to do that.